mvp-spec/model-updater/generate-model-markdown.sh

```diff
@@ -1,16 +1,29 @@
 #sh
 
 BASEDIR=$(dirname "$0")
+TMP_DIR="tmp-schemas"
 
 # Because generate-schema-doc only supports relative paths
 cd "$BASEDIR" || exit
 
+# Move all the schemas in a temporary directory
+# It allows to fetch schemas from different directory
```

Comment:
I think it is a good idea to allow schemas to be in separate dirs.
However, with this implementation the schemas themselves are not valid: `file transmission/seed.json` references `"version.json"` but this reference is incorrect because the path is not valid. It should be `../version.json`. The issue can be seen with an IDE that supports JSON schemas validation, for instance. (I'm using WebStorm with JSON schema plugin).
I think an alternative would be to use the proper reference (`../version.json`) and adjust this script to copy files while preserving directories. And then make sure the generation of markdown supports subdirs.

Reply:
I think that the current directories of `partials` and `json-schemas`has two much things.I would like to mitigate it with different strategies.  One of them is sorting partials and json-schemas in directories. I tried to use relative references in the schemas and it failed. So I prefer to simplify it with this temporary directory in the script that gather everything.

I prefer to merge this approach. Then we can sort everything incrementally without having to change the relative pathes each time. Once all the sorting is done, then we can retry relative pathes.

Reply:
As I said, as they are, these JSON Schema files are **invalid**, in the sense that the other schemas they are referring to are not found. I think this is an issue to introduce JSON schemas that are _invalid from the beginning_.

> I tried to use relative references in the schemas and it failed.
We should investigate what went wrong and fix it. Unless there is a hard point that we can't fix in a reasonable time, in which case I'm ok to first merge this PR and then fix it, but it would be a pity.

Reply:
As I said, as they are, these JSON Schema files are **invalid**, in the sense that the other schemas they are referring to are not found. I think this is an issue to introduce JSON schemas that are _invalid from the beginning_.

> I tried to use relative references in the schemas and it failed.
We should investigate what went wrong and fix it. Unless there is a hard point that we can't fix in a reasonable time, in which case I'm ok to first merge this PR and then fix it, but it would be a pity.

Reply:
They are valid when their are preprocessed so it is really not a big deal. We may even have the best of the both words: simple reference + directories.

Reply:
Of course they become valid after processing, but my point is that these files are not valid on the repo, which means if someone wants to use them to generate a new API or simply navigate through them, they won't be able. As a contributor, I cannot navigate through these files or generate code from them.
I think it's a great idea to organise the files into subdirectories, but there is no rush to do it so we should only do it when we have a working solution. I still don't understand what was the issue with using relative paths.

Reply:
Ok I have forgotten that it was used for DTO. In this case, I agree with you.
Today, I tried again to use relative paths. It still doesn't work. To move forward quickly, I move all the files in the root directory.